EntranceRandomizer.py

```diff
@@ -79,7 +79,11 @@ def defval(value):
                              Triforce Hunt: Places 30 Triforce Pieces in the world, collect
                                             20 of them to beat the game.
                              Local Triforce Hunt: Places 30 Triforce Pieces in your world, collect
-                                            20 of them to beat the game.       
+                                            20 of them to beat the game.
+                             Ganon Triforce Hunt: Places 30 Triforce Pieces in the world, collect
+                                            20 of them, then defeat Ganon.
+                             Local Ganon Triforce Hunt: Places 30 Triforce Pieces in the world,
```

Comment:
local triforce hunt says "in your world", this wasn't copied to local ganon triforce hunt

Reply:
Resolved in d843119